repository-data/webfiles/src/main/resources/site/freemarker/hee/macros/quick-links.ftl

```diff
@@ -4,26 +4,31 @@
 
 <#macro quickLinks quickLinks>
     <#if quickLinks?? && quickLinks.links?? && quickLinks.links?size != 0>
+        <#assign hasQuickLinkContent=quickLinks.links?filter(link -> link.text?has_content)?has_content>
         <div class="nhsuk-grid-column-one-third">
-            <div class="nhsuk-card">
-                <div class="nhsuk-card__content">
-                    <#if quickLinks.title?has_content>
-                        <h3 class="nhsuk-card__heading">${quickLinks.title}</h3>
-                    </#if>
+            <#if quickLinks.title?has_content || hasQuickLinkContent>
```

Comment:
@thojava - Do you think it needs to be `<#if quickLinks.title?has_content && hasQuickLinkContent>` ? Not sure a title without any links is worth rendering.

Reply:
@smanivasag I see your point. But I think that when editor try out this component, he may want to put just the title and see how the component being rendered then continue adding the links. So I think showing just the title is better than a blank.
I am not sure that is too supportive to the editor ?

Reply:
Sure, that makes sense from the editor perspective 👍